app/models/resource.rb

```diff
@@ -2,43 +2,43 @@
 
 class Resource < Sequel::Model
   include HasId
-  
+
   unrestrict_primary_key
-  
+
   one_to_many :permissions, reciprocal: :resource
   one_to_many :annotations, reciprocal: :resource
   one_to_many :secrets, reciprocal: :resource
   one_to_many :policy_versions, reciprocal: :resource, order: :version
   one_to_many :host_factory_tokens, reciprocal: :resource
   many_to_one :owner, class: :Role
   many_to_one :policy, class: :Resource
-  
+
   alias id resource_id
-  
+
   def kind
     id.split(":", 3)[1]
   end
-  
+
   def identifier
     id.split(":", 3)[2]
   end
-  
+
   def as_json options = {}
     super(options).tap do |response|
       response["id"] = response.delete("resource_id")
       %w(owner policy).each do |field|
         write_id_to_json response, field
       end
-      response["permissions"] = permissions.as_json.map {|h| h.except 'resource'}
-      response["annotations"] = self.annotations.as_json.map {|h| h.except 'resource'}
+      response["permissions"] = permissions.as_json.map { |h| h.except 'resource' }
+      response["annotations"] = self.annotations.as_json.map { |h| h.except 'resource' }
       case kind
       when "variable"
         response["secrets"] = secrets_dataset.order(:version).as_json
-          .map { |h| h.except 'resource' }
+                                             .map { |h| h.except 'resource' }
```

Comment:
can you fix all the non-code changes that you made to this file?

Reply:
Talked about it